ports/hello-imgui/portfile.cmake

```
@@ -0,0 +1,164 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
```

Is this restriction necessary outside windows? (I don't know.)
```suggestion
if(VCPKG_TARGET_IS_WINDOWS)
    vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
endif()
```